bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/handler/NanoleafControllerHandler.java

```diff
@@ -12,10 +12,33 @@
  */
 package org.openhab.binding.nanoleaf.internal.handler;
 
-import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.*;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.API_ADD_USER;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.API_DELETE_USER;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.API_EFFECT;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.API_EVENTS;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.API_GET_CONTROLLER_INFO;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.API_MIN_FW_VER_LIGHTPANELS;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.API_RHYTHM_MODE;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.API_SET_VALUE;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.BRIGHTNESS_STEP_SIZE;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CHANNEL_COLOR;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CHANNEL_COLOR_MODE;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CHANNEL_COLOR_TEMPERATURE;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CHANNEL_COLOR_TEMPERATURE_ABS;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CHANNEL_EFFECT;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CHANNEL_RHYTHM_ACTIVE;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CHANNEL_RHYTHM_MODE;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CHANNEL_RHYTHM_STATE;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CHANNEL_SWIPE;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CHANNEL_SWIPE_EVENT_DOWN;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CHANNEL_SWIPE_EVENT_LEFT;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CHANNEL_SWIPE_EVENT_RIGHT;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CHANNEL_SWIPE_EVENT_UP;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.CONTROLLER_PANEL_ID;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.DEVICE_TYPE_LIGHTPANELS;
+import static org.openhab.binding.nanoleaf.internal.NanoleafBindingConstants.DEVICE_TYPE_TOUCHSUPPORT;
```

Comment:
I think wildcard imports are allowed for static imports of constants, just not for classes ;)

Reply:
I wished I had known that before 😃